src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -385,6 +385,33 @@
           </div>
         </fieldset>
 
+        <!-- Tags -->
+        <fieldset *ngIf="editing">
+          <legend class="cd-header"
+                  i18n>Tags
```

Comment:
A helper would be nice to know what tags represents here,

Reply:
Sure, I'll add it to explain the Tags function

Reply:
Added a helper to provide some context